docs/components/authentication/google-one-tap.mdx

```diff
@@ -0,0 +1,263 @@
+---
+title: <GoogleOneTap /> component
+description: Clerk's <GoogleOneTap /> component renders a UI for authenticating users with Google's One Tap API.
+---
+
+# `<GoogleOneTap />` component
+
+> To use Google One Tap with Clerk, you must [enable Google as a social connection in the Clerk Dashboard](https://clerk.com/docs/authentication/social-connections/google#configuring-google-social-connection) and make sure to use custom credentials.
+
+The `<GoogleOneTap />` component renders the [Google One Tap](https://developers.google.com/identity/gsi/web/guides/features) UI so that users can use a single button to sign-up or sign-in to your Clerk application with their Google accounts.
+
+By default, this component will redirect users back to the page where the authentication flow started, however you can override this with [force redirect URL props](#google-one-tap-props) or [force redirect URL environment variables](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects).
+
+## `GoogleOneTapProps`
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `cancelOnTapOutside?` | boolean | If true, the One Tap prompt closes automatically if the user clicks outside of the prompt. Default: `true`. |
+| `itpSupport?` | boolean | If true, enables the [ITP-specific UX](https://developers.google.com/identity/gsi/web/guides/itp) when One Tap is rednered on ITP browsers such as Chrome on iOS, Safari, and FireFox. Default: `true`. |
+| `fedCmSupport?` | boolean | If true, enables Google One Tap to use [the FedCM API](https://developers.google.com/privacy-sandbox/3pcd/fedcm) to sign users in. See Google's docs on [best practices when disabling FedCM support](https://developers.google.com/identity/gsi/web/guides/display-google-one-tap#do_not_cover_google_one_tap). Default: `true` |
+| `signInForceRedirectUrl?` | `string` | Useful if you want to redirect to a path specific to Google One Tap users. If provided, this URL will **always** be redirected to after the user signs in, overriding any [`ClerkProvider` redirect URL props](/docs/components/clerk-provider#properties) or [redirect URL environment variables](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects). |
+| `signUpForceRedirectUrl?` | `string` | Useful if you want to redirect to a path specific to Google One Tap users. If provided, this URL will **always** be redirected to after the user signs up, overriding any [`ClerkProvider` redirect URL props](/docs/components/clerk-provider#properties) or [redirect URL environment variables](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects). |
+
+## Usage with frameworks
+
+The following example includes basic implementation of the `<GoogleOneTap />` component. You can use this as a starting point for your own implementation.
+
+<Tabs type="framework" items={["Next.js"]}>
+  <Tab>
+
+    ```tsx filename="app/page.tsx" {2, 15}
```

Comment:
I don't think this example is correct - this looks like the `layout.tsx` but the file is called `app/page.tsx`.
I would expect this code to be in the layout, and this `page.tsx` example to look like the other component examples where there is a home page and it renders the googleonetap sign in button. See https://clerk.com/docs/components/authentication/sign-in